3D-microservices-observability/node-js-app/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "beachshirt",
+  "version": "1.0.0",
+  "description": "beachshirt application which makes cool shirts for the beach.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Yogesh Prasad",
+  "license": "Apache-2.0",
```

Comment:
Please fill in other relevant fields: https://docs.npmjs.com/files/package.json

At a minimum, `homepage`, `keywords` and `bugs` props